Pod/UIKit/Classes/UILabel+BonMotUtilities.m

```diff
@@ -7,33 +7,35 @@
 //
 
 #import "BonMot.h"
+#import "BONChain_Private.h"
+
 @import ObjectiveC.runtime;
 
 @implementation UILabel (BonMotUtilities)
 
-- (void)setBonTextable:(id<BONTextable>)textable
+- (void)setBonChain:(BONChain *)chain
 {
-    objc_setAssociatedObject(self, @selector(bonTextable), textable, OBJC_ASSOCIATION_COPY_NONATOMIC);
+    objc_setAssociatedObject(self, @selector(bonChain), chain, OBJC_ASSOCIATION_COPY_NONATOMIC);
 
-    // If the textable is empty, use the label’s existing text
-    if (textable.text.generatesEmptyString) {
+    // If the chain is empty, use the label’s existing text
+    if (chain.generatesEmptyString) {
```

Comment:
Perhaps check if the old chain had a string that's equal to the text? If it did, clear the text. Otherwise, leave the text alone as it was likely assigned via one of the text properties directly.